update.sh

```diff
@@ -17,7 +17,7 @@ echo "Updating certificates..."
 ./certs/update.sh
 
 echo "Fetching and building traefik $VERSION..."
-wget -O traefik https://github.com/emilevauge/traefik/releases/download/$VERSION/traefik
+wget -O traefik https://github.com/containous/traefik/releases/download/$VERSION/traefik
```

Comment:
While on it, could you also please add `--fail`? This is needed if `set -e` is supposed to be effective here.